samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -71,12 +71,27 @@
     /* "Key": // dotnet user-secrets set "AzureSpeech:Key" "MY_AZURE_SPEECH_KEY" */
   },
 
+  "Auth": {
+    //
+    // Auth to use to gate access to exposed API's.
+    // - Supported Types are "None", "ApiKey", or "AzureAd".
+    //
+    "Type": "None",
+    "ApiKey": "", // The API key to use for auth - use user secrets, environment variable or Key Vault to populate
```

Comment:
If we want to use the same pattern here as we do for the other Key properties, let's comment it out and display a "dotnet user-secrets" comment next to it like we do elsewhere in this file.

Passing thought: Maybe there's an opportunity for a small script to automatically generate and set the API key in both webapi and webapp.

Reply:
Done. 

I'm looking into automatic deployment for SKaaS.  From that, it's just a small jump to automate the setup of Copilot Chat... More to follow